src/pybind/mgr/orchestrator.py

```diff
@@ -5,10 +5,18 @@
 Please see the ceph-mgr module developer's guide for more information.
 """
 
+try:
+    from typing import TypeVar, Generic, List
+    T = TypeVar('T')
+    G = Generic[T]
+except ImportError:
+    T, G = object, object
 
-class _Completion(object):
+
+class _Completion(Generic[T]):
```

Comment:
This probably needs to be `class _Completion(G)` for python 2 compatibility.

Reply:
fixed